README.md

```diff
@@ -16,6 +16,13 @@ git clone https://github.com/contentful/boilerplate-php.git
 
 Open your terminal, navigate to this folder and run `./run.sh`. That's it!
 
+### To run from CLI
```

Comment:
Hey,

I am not 100% sure, but I see the same calls in the `./run.sh`. Should we maybe only have one recommendation on how to setup the boilerplate from a cli?

Reply:
Ok.  I will remove this comment as it may be confusing with the contentful cli.